src/knoop/Node.hpp

```diff
@@ -1,10 +1,10 @@
 template< typename... Ts >
 using void_t = void;
 
-template< typename... Ls >
+template< typename L, typename... Ls >
```

Comment:
Does this make it such that one must provide at least one template parameter?

Reply:
That is correct. Note that this adds no new restrictions, because at least one template parameter already had to be provided before my change - because std::variant, on which Node is based, required at least one. Now it's explicit, and a use error occurs with Node (less cryptic) that inside the variant (more cryptic).